src/pages/user/components/user-layout.tsx

```diff
@@ -0,0 +1,82 @@
+import {FireIcon, BookmarkIcon, UserCircleIcon} from '@heroicons/react/outline'
+import cx from 'classnames'
+import Link from 'next/link'
+import {useRouter} from 'next/router'
+import {useViewer} from 'context/viewer-context'
+import LoginRequired from 'components/login-required'
+
+const userPagesMap = [
+  {
+    path: '/user/account',
+    name: 'Account',
+    icon: UserCircleIcon,
+  },
+  {
+    path: '/user/activity',
+    name: 'Activity',
+    icon: FireIcon,
+  },
+  {
+    path: '/user/bookmarks',
+    name: 'Bookmarks',
+    icon: BookmarkIcon,
+  },
+]
+
+export default function UserLayout({children}: any) {
+  const {viewer} = useViewer()
+  const router = useRouter()
+  const currentPath = router.asPath
+
+  return (
+    <div className="">
+      <LoginRequired>
+        <main className="">
+          <div className="max-w-screen-xl mx-auto px-4 pb-6 sm:px-6 lg:px-8 lg:pb-16">
+            <div className="sm:px-6 md:px-0 pt-10 pb-4">
+              <h1 className="text-3xl font-bold tracking-tight text-gray-900">
+                Hello, {viewer?.full_name || viewer?.username}!
+              </h1>
+            </div>
+            <div className="overflow-hidden rounded-lg bg-white shadow">
+              <div className="divide-y divide-gray-200 lg:grid lg:grid-cols-12 lg:divide-y-0 lg:divide-x">
+                <aside className="py-6 lg:col-span-3">
+                  <nav className="space-y-1">
+                    {userPagesMap.map((page) => {
+                      return (
+                        <Link key={page.path} href={page.path}>
+                          <a
+                            className={cx(
+                              page.path === currentPath
```

Comment:
the only weird thing about styling here is if a member goes to pricing page, the route will still be `/pricing` so the menu item won't be highlighted